files/kubelet.service

```diff
@@ -0,0 +1,22 @@
+[Unit]
```

Comment:
please add a comment like `# THIS FILE IS MANAGED BY PUPPET`

Reply:
done